package.json

```diff
@@ -5,10 +5,11 @@
   "homepage": "https://github.com/awslabs/aws-crt-nodejs",
   "repository": "github:awslabs/aws-crt-nodejs",
   "contributors": [
-    "Colden Cullen <colden@amazon.com>"
+    "AWS SDK Common Runtime Team <aws-sdk-common-runtime@amazon.com>"
```

Comment:
I'm 90% sure this alias won't accept emails from external addresses, so maybe it doesn't go here

Reply:
We use it on all of our other packages :)

Reply:
SHIP IT